documentation/howtos.md

```diff
@@ -16,14 +18,6 @@ EDXAPP_ADDITIONAL_CRON_JOBS:
   hour: "*"
   minute: "43"
   day: "*"
-# update_index is required but isn't currently part of the provisioning process. So in the interests of saving time, this is run every 10 minutes through cron (it's not an expensive operation).
```

Comment:
FYI @swalladge , I moved this command down to a [one-off manual step](https://github.com/open-craft/opencraft/pull/554/commits/9f12462b7d4f7a746f86cd547a4dff21382b5094#diff-e62c375eb49e96b787ac278c5d78e70dR165), since we're now aware that we need to run ES on a separate VM.

Reply:
Looks good. Could you  also clarify that this command is run on the appserver, and it's a once-off thing because the ES server is expected to be on a long-lived dedicated instance?

Reply:
I don't think we need to clarify that at this line, since we're removing it entirely?  And I think [this line under Manual Steps](https://github.com/open-craft/opencraft/pull/554/commits/9f12462b7d4f7a746f86cd547a4dff21382b5094#diff-e62c375eb49e96b787ac278c5d78e70dR90) indicates that these are one-off steps.. does it need further clarification?

Reply:
Yep I meant the clarifications should go where the manual step is documented. It wasn't really clear to me that this command is to be run on the openedx appserver, and that the ES server is expected to be a dedicated server.

Reply:
Gotcha, I see why that was confusing.  Is 71c62ce  better?

Reply:
LGMT :+1: